clustermesh/clustermesh.go

```diff
@@ -2285,7 +2288,7 @@ func removeFromClustermeshConfig(values map[string]interface{}, clusterName stri
 	if !ok {
 		return nil, fmt.Errorf("existing clustermesh.config.clusters array is invalid")
 	}
-	oldClusters := make([]map[string]any, 0, len(cs))
+	outputClusters, victim := make([]map[string]any, 0, len(cs)), false
```

Comment:
It seems to me that these changes (and the ones below) should be part of the other commit.